src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostOptions.cs

```diff
@@ -29,5 +29,22 @@ internal static class RemoteHostOptions
 
         [ExportOption]
         public static readonly Option<bool> RemoteHostTest = new Option<bool>(nameof(InternalFeatureOnOffOptions), nameof(RemoteHostTest), defaultValue: false);
+
+        public const int CrashVS = 0;
+        public const int ShowInfoBarAndRunFeaturesInProcIfPossible = 1;
+        public const int ShowInfoBarAndDisableAllFeatures = 2;
+
+        // Define VS behavior on unintentional OOP shutdown such as user killing OOP process
+        //
+        // CrashVS: 
+        //      crash VS
+        // ShowInfoBarAndRunFeaturesInProcIfPossible: [default]
+        //      show info bar and let VS keep running. some features might keep running inproc based on how they are implemented.
+        // ShowInfoBarAndDisableAllFeatures: 
+        //      show info bar and disable all features.
```

Comment:
The default will be decided at Shiproom today, though of course based on our recommendation.  Sri prefers 2, I prefer 3.

And yes, new telemetry was supposed to be a part of this -- both NFW and a telemetry point with the relevant info.